app/demo-stm32g0-nucleo/Cargo.toml

```diff
@@ -16,7 +16,7 @@ cortex-m-rt = { workspace = true }
 cfg-if = { workspace = true }
 stm32g0 = { workspace = true, features = ["rt"] }
 
-kern = { path = "../../sys/kern" }
+kern = { path = "../../sys/kern", features = ["no-dump"] }
```

Comment:
This seems to violate the "features should be additive" rule of thumb; however, changing it would require editing a bunch of TOML files to add `features = ["dump"]`...

Reply:
Yeah, it *does* violate the rule of thumb, but it's also unfortunate to require it ever TOML simply because we are so extraordinarily flash constrained on the G0s.  How important is the rule of thumb?

Reply:
It wouldn't be the worst abuse of features in Hubris; I'm fine letting it stand unless anyone else is especially offended.

Reply:
Agreed; flipped this around and then added `dump` as a default feature.